src/app/view/components/_common/dot-global-message/dot-global-message.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "primeng-theme/utils/theme-variables";
+
+:host{
+    display: flex;
+
+    div:first-child{
```

Comment:
You don't need to nest everything under host... that's why BEM class name are for.

Also the angular encapsulation takes care of this.

Fix the spacing.

You don't need ```div.xxx```